typings/test.ts

```diff
@@ -1,14 +1,14 @@
 // This is a test file for the TypeScript typings.
 // It is not intended to be used by external users.
-import Telegraf, { Markup } from './index';
+import Telegraf, { Markup, Middleware, ContextMessageUpdate } from './index';
 
 
 const randomPhoto = 'https://picsum.photos/200/300/?random'
-const sayYoMiddleware = ({ reply }, next) => reply('yo').then(() => next())
+const sayYoMiddleware: Middleware<ContextMessageUpdate> = ({ reply }, next) => reply('yo').then(() => next && next())
```

Comment:
This fixes:
  * error TS7031: Binding element 'reply' implicitly has an 'any' type
  * error TS7006: Parameter 'next' implicitly has an 'any' type
  * error TS2722: Cannot invoke an object which is possibly 'undefined' (the definition of `Middleware` says that `next` is optional and can be `undefined`)